front_end/ndb/NdbMain.js

```diff
@@ -55,10 +55,19 @@ Ndb.NdbMain = class extends Common.Object {
     await Ndb.nodeProcessManager.addFileSystem(environment.cwd);
   }
 
-  async _startRepl() {
-    const environment = await Ndb.environment();
-    Ndb.nodeProcessManager.debug(await Ndb.nodeExecPath(), [environment.repl])
-        .then(this._startRepl.bind(this));
+  async _repl() {
+    /**
+     * Buffer.from(`console.log('Welcome to the ndb %cR%cE%cP%cL%c!',
+     * 'color:#8bc34a', 'color:#ffc107', 'color:#ff5722', 'color:#2196f3', 'color:inherit');
+     * setInterval(_ => 0, 2147483647)//# sourceURL=repl.js`).toString('base64')
+     */
+    const code = 'Y29uc29sZS5sb2coJ1dlbGNvbWUgdG8gdGhlIG5kYiAlY1IlY0UlY1AlY0wlYyEnLAonY29sb3I6IzhiYzM0YScsICdjb2xvcjojZmZjMTA3JywgJ2NvbG9yOiNmZjU3MjInLCAnY29sb3I6IzIxOTZmMycsICdjb2xvcjppbmhlcml0Jyk7CnNldEludGVydmFsKF8gPT4gMCwgMjE0NzQ4MzY0NykvLyMgc291cmNlVVJMPXJlcGwuanM=';
+    const args = ['--title=repl',
+        '-e', `eval(Buffer.from('${code}', 'base64').toString())`];
+    const options = { ignoreOutput: true };
+    const node = await Ndb.nodeExecPath();
+    for (;;)
```

Comment:
This will keep biting us, can we detect exit and restart on demand?

Reply:
We need event from DevTools: user is typing something in console but we do not have available target.